javascript-practice/src/styles/bases/reset.css

```diff
@@ -168,3 +168,7 @@ input,
 select {
   vertical-align: middle;
 }
+
+button {
+  cursor: pointer;
+}
```

Comment:
Should add empty line at the end of files. Pls check all

Reply:
I done